app/views/dishes_templates/index.html.erb

```diff
@@ -103,7 +103,7 @@
                   <% if current_user.bookmarks.where("dish_id ="+dishes.id.to_s)[0] != nil%>
                     <a href="/venues/<%=dishes.specialists[0].id%>">
                       <%= current_user.bookmarks.where("dish_id ="+dishes.id.to_s)[0].venue.name %>
-                      
+                  
```

Comment:
Weird whitespace thing, but I'm not going to worry about it. We can fix formatting/whitespace at the end.